modules/claims_api/app/controllers/claims_api/v2/veterans/power_of_attorney/request_controller.rb

```diff
@@ -59,6 +59,15 @@ def validate_accredited_organization(poa_code)
           @organization = ::Veteran::Service::Organization.find_by(poa: poa_code)
         end
 
+        def build_bgs_attributes(form_attributes)
```

Comment:
Rubocop was upset with the controller method length so I moved this chunk of code